assets/app/actions/createDraftBranchName.js

```diff
@@ -0,0 +1,5 @@
+import { encodeB64 } from '../util/encoding';
+
+export default path => {
```

Comment:
You are absolutely correct these one-off functions are not actions-- they are in this directory not because they share the same class of functionality as the "real actions" in this directory but because I didn't put them in a different directory when I extracted them. I didn't immediately have a good name for where to put them (utility seems a bit vague and grab-baggy) so I left them there.

I can be very lazy.

I'll put them somewhere and take a swing at better naming/organization.